contents/handbook/company/team/james-hawkins.md

```diff
@@ -49,30 +49,33 @@ I tend to work 9am to 5pm with an hour for lunch, then I have a gap to have dinn
 
 ## How you can help me
 
-- Come to 1:1s with an agenda and clear things I can help with.
+- Come to 1:1s with an agenda, energy and things I can help with.
 - Ship :) Lots of things will be on fire in a startup all the time. Make sure you're moving the needle every week.
 - Due to my role, it's easy for me to accidentally steer you. Sometimes this means your team will get confused. Be communicative back to your team if I've changed your direction. And you don't have to listen to me.
 
-## My goals until end December 2022
+### Personal strategy as of Q1 2023
 
-We hit product market fit and are now seeing accelerating adoption in high quality companies.
-
-I need to make sure we maintain a high month over month revenue growth rate whilst staying default alive, since this will make future fundraising and expansion easy.
-
-### Personal strategy
-
-- Make sure every team reaches an ["Innovating" state](https://danlebrero.com/2022/07/06/an-elegant-puzzle-systems-of-engineer-management-book-summary/)
-- Support the marketing team with my individual work 
+- make sure every team reaches an ["Innovating" state](https://danlebrero.com/2022/07/06/an-elegant-puzzle-systems-of-engineer-management-book-summary/)
+- maintain a high month over month revenue growth rate whilst staying default alive, since this will make future fundraising and expansion easy
```

Comment:
Agreed this is 1000% important, though I'm not _super_ sure what this means for you personally? (Given this is our top company objective as well)

Reply:
(Also caps)

Reply:
removed the revenue one - as actually i'm trying to give the growth team more space